### DISCUSSION TITLE
S3 upload failure [Apr 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1982] [Discord Threads]
> 🙋‍♂️ **silverlight**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### hi. please create an issue in github  
#  
> 🙋‍♂️ **silverlight**  
#### thanks <@495466997304262658>  
#  
> 🙋‍♂️ **silverlight**  
#### we already had an issue here https://github.com/nocodb/nocodb/issues/1822  
#  
> 👨‍🚀 **wingkwong**  
#### oh this one. will take a look later  
#  
> 🙋‍♂️ **silverlight**  
#### thanks <@495466997304262658>  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/968700322677686282)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
